README.md

```diff
@@ -32,5 +32,5 @@ Open Software License \("OSL"\) v3.0
 
```

Comment:
@gary-nosto which license is Overdose plugin published under?

Reply:
I just asked Overdose about this.

Reply:
Removed license.